src/markdown-pages/why-galasa.md

```diff
@@ -0,0 +1,124 @@
+
+Once written, a Galasa test is available 24x7 for reuse.
+</details>
+
+## Centralizing, maintaining and storing results consistently:
+
+<details>
+<summary><b>Maintaining, recording and planning with a large suite of tests</b></summary>
+
+Maintaining a set of test materials (which could be written in several different languages), recording which tests have been run, and scheduling any outstanding tests is difficult and time consuming. Manual tests are often split across teams and reported separately, with no single, consistent view of the testing.
+
+The Galasa test catalog provides the ability to generate a clear description of the areas covered by any given test. Related tests can be stored within a shared test catalog, from which tests can be automatically selected to run for any given change set. Automated regression test suites can be created for new software versions, so you can run a specified set of tests for automated baselining of a new environment installation, such as a hardware migration. 
+</details>
+
+<details>
+<summary><b>Diagnosing failures </b></summary>
+
+Investigating test failures takes up time and can be particularly tedious with larger scripts. Test artifacts are often stored in lots of different repositories, making it time-consuming and difficult to locate the right information to help you root out the cause of a failure.
+
+Galasa automatically stores all test artifacts in a single, central repository, making diagnostics quicker and easier and allowing big picture information to be extracted easily. You can also debug tests using a local instance of Galasa, so you can examine every line of code.
+</details>
+
+<details>
+<summary><b>Big picture communication</b></summary>
+
+Test results are often stored in spreadsheets and manually approved by product owners before changes are promoted. This makes it difficult to understand the tests that have been run, and the manual intervention required as part of the sign-off process can delay delivery.
+
+Galasa's dashboard integrates all your test results in one place, making reporting and reviewing between test phases easy and consistent.
+</details>
+
+## Extensibility and scalability - future-proofing your testing:
+
+<details>
+<summary><b>Maintaining performance levels</b></summary>
+
+Running scripts and tooling on tools such as Jenkins can lead to performance issues – ideally the tests and the Jenkins nodes should not share workload. Whilst you don’t want to miss defects by limiting the number of tests run, the more tests that are running the longer the testing phase takes, and the increased amount of computer resources that are required from the Jenkins node will limit the throughput.
+
+Galasa’s tests as a service allows the test workload to be moved off a Jenkins node, with the added benefit of being capable to scale horizontally in its own cloud environment. 
```

Comment:
Another oppurunity to mention galasa's resource locking preventing any tests from colliding, instead queue tests until the resources are free?